### ISSUE TITLE
Swift/Obj-c snippet conversions crash when given empty/invalid form data

### ISSUE THREAD

**Issue by pimterry** [Kong/httpsnippet#161]
When converting requests with a `application/x-www-form-urlencoded` content type, if there are no successfully parsed parameters in the body, the Swift & Obj-C conversions both crash with "Cannot read property 'name' of undefined". That happens because these lines both assume that there's at least one parameter:

https://github.com/Kong/httpsnippet/blob/master/src/targets/swift/nsurlsession.js#L49

https://github.com/Kong/httpsnippet/blob/master/src/targets/objc/nsurlsession.js#L49

Most (all?) cases where you'll hit this require technically invalid bodies, e.g. sending empty or JSON bodies as form data, but unfortunately that's not uncommon. It would be better to not crash in this case, as in every other target.

**Comment by dimitropoulos** [comment]
I'm almost certain this has been fixed now.  Today, thanks to TypeScript, this code has been updated to wrap with `if (postData.params) {` which will prevent calling `.name` on a param value.

If I'm mistaken, please show some repro steps and we'd be happy to take a look!

**Comment by pimterry** [comment]
This hasn't been fixed. Repro:

* Download [this HAR file](https://gist.github.com/pimterry/076102da7c03f9e851a3b69613379934) into the root of your HTTP Snippet folder
* In Node, run:
    ```javascript
    > har = require('./empty-params.har.json')
    > HS = require('.')
    > hs = new HS.HTTPSnippet(har)
    > hs.convert('swift', 'nsurlsession')
    Uncaught TypeError: Cannot read property 'name' of undefined
        at convert (.../dist/targets/swift/nsurlsession/client.js:75:82)
        at .../dist/httpsnippet.js:238:74
        at Array.map (<anonymous>)
        at HTTPSnippet.convert (.../dist/httpsnippet.js:238:42)
    ```

(Note that this requires the fix in #294 because otherwise you currently can't require httpsnippet in node at all).

I'll open a PR to fix it in a sec.
